Extensions/Program.cs

```diff
@@ -1,2 +1,50 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿/// <summary>
+/// Why extension methods? Extension methods enable you to "add" methods to existing types without creating a new derived type
+/// The extension methods are static methods, but they're called as if they were instance methods on the extended type. 
+/// 
+/// When using an extension method to extend a type whose source code you aren't in control of, you run the risk that a change in the implementation of the type will cause your extension method to break.
+/// 
+/// 
```

Comment:
remove whitespace

Reply:
I'll go ahead and remove the whitespace

Reply:
I will go ahead and remove the whitespace

Reply:
New comment